R/visualize_embeddings.R

```diff
@@ -143,77 +144,150 @@ keep_tokens <- function(embedding_df, tokens = "[CLS]") {
 #'     display_pca()
 #' }
 display_pca <- function(embedding_df,
+                        project_vectors = embedding_df,
                         color_field = NULL,
                         disambiguate_tokens = TRUE,
                         hide = NULL) {
```

Comment:
I say nuke `hide` completely and mention that in the news.

Reply:
ok, sounds good